sdk/maps/azure-maps-render/src/main/java/com/azure/maps/render/MapsRenderClient.java

```diff
@@ -224,7 +224,7 @@ public Response<MapAttribution> getMapAttributionWithResponse(TilesetId tilesetI
      * <p>
      * Fetches state tiles in vector format typically to be integrated into indoor maps module of map control or SDK.
      * The map control will call this API after user turns on dynamic styling (see [Zoom Levels and Tile
-     * Grid](https://docs.microsoft.com/azure/location-based-services/zoom-levels-and-tile-grid)).
+     * Grid](<a href="https://docs.microsoft.com/azure/location-based-services/zoom-levels-and-tile-grid">...</a>)).
```

Comment:
What's this? Why it's `...` for the text?

Reply:
It was suggested form javadoc warning, now I added text for this link. @dubiety

Reply:
Sorry, what text? I'm a little bit confused

Reply:
@dubiety text inside <a> tag